### PR TITLE
test(storybook): support running visual tests in Chromatic

### LINKED ISSUE #963: Test Chromatic for visual testing
We have had several bugs due to lack of regression visual regression testing. This will just get worse as component library grows and we make a css library.

Investigate on what options we have in this regard.

Notes:
- https://www.chromatic.com/
- https://docusaurus.io/blog/upgrading-frontend-dependencies-with-confidence-using-visual-regression-testing
- https://percy.io/
- https://loki.js.org/

### PR DESCRIPTION
**Pull request by unekinn**
Closes #963

To better make use of visual tests:
- "Avatar/In Dropdown" story renders with the dropdown open
- Modals, comboboxes, dropdowns, and popovers are now opened
  automatically (when not in docs) through interaction tests
- SkipLink: added a story showing the SkipLink when it has become visible
- Switch: added examples of enabled and hovered switches
- Added a default 1rem padding to stories, to account for normally
  overflowing content like focus styles, floating badges etc
- Several stories have custom styling to ensure elements that have been
  removed from the normal layout flow (e.g. with absolute positioning)
  or are rendered outside the Story root element (e.g. with a React
  portal) are actually visible in the snapshots. To easier facilitate
  this, a global customStylesDecorator has been added, which adds styles
  configured through `parameters.customStyles`.

This commit also replaces story-specific decorators which only added
styling with `parameter.customStyles`, simplifying the stories a bit.

Visual tests on 320px wide screen revealed a bug where Combobox was
wider than the screen. This has been fixed.

With the changes that open modals, add pseudo states etc, some new
accessibility violations surfaced.
- Fixed axe violation `svg-img-alt` in Dropdown stories
- Disabled axe rule `color-contrast` when the pseudo-class :active is
  emulated through `storybook-addon-pseudo-states`, since we concluded
  that 4.5:1 text contrast during press actions is unnecessary

Chromatic is run automatically for pull requests through Github Actions
-- see `.github/workflows/test.yml`.

If you want to trigger visual tests from your own machine, add (or edit)
the file `apps/storybook/.env`. This file is in .gitignore and will not be
committed.

Add the following to `.env`:
```
CHROMATIC_PROJECT_TOKEN=<token>
```

Replace `<token>` with the token found [here](https://www.chromatic.com/manage?appId=66fe736b9d639fe6801bf130&setup=true), under "Setup Chromatic with this project token".

Then run these commands:
```
yarn build:storybook
yarn chromatic
```

You can also replace the last command with e.g.
```
yarn chromatic --no-only-changed --only-story-names "Komponenter/Modal/*"
```
...to only run tests for the Modal components

### CODE REVIEW

File: chromatic.config.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://www.chromatic.com/config-file.schema.json",
+  "onlyChanged": true,
```
Comment (unekinn) [inline chromatic.config.json:3]: Eg er usikker på om `onlyChanged` kanskje må settast `false` eller at vi må legge inn `externals: ["packages/css", "pachages/theme"]`, for å sørge for at visuelle testar blir kjørt på nytt ved endringer i css og tema. Tenker å teste dette etter vi har fått inn baseline snapshots, før den tid vil alle stories lage nye snapshots uansett

File: packages/react/src/components/form/Combobox/Combobox.stories.tsx [hunk 2]
```diff
@@ -16,18 +17,47 @@ import { Combobox } from './index';
...
+  parameters: {
+    layout: 'fullscreen',
+    customStyles: {
+      maxWidth: '30rem',
+      story: {
+        height: '340px',
+      },
+    },
+    a11y: {
+      // TODO: this rule should be enabled after figuring out why floating-ui sets aria-hidden
```
Comment (unekinn) [inline packages/react/src/components/form/Combobox/Combobox.stories.tsx:29]: Should fix this comment to not single out floating-ui & aria-hidden

File: .changeset/slimy-buttons-train.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@digdir/designsystemet-css': patch
+---
+
+Fixed Combobox overflowing the screen width on very narrow screens (~340px and less)
```
Comment (Barsnes) [inline .changeset/slimy-buttons-train.md:5]:
```suggestion
Combobox: Fix overflowing screens ~340px or less
```

Comment (unekinn) [reply .changeset/slimy-buttons-train.md]: Oppdatert.

**Pull request merged**